app/javascript/src/question_radios.js

```diff
@@ -15,25 +15,23 @@
  *
  **/
 
-
-const utilities = require('./utilities');
-const mergeObjects = utilities.mergeObjects;
-const safelyActivateFunction = utilities.safelyActivateFunction;
-const createElement = utilities.createElement;
-const uniqueString = utilities.uniqueString;
-const Question = require('./question');
-const ActivatedMenu = require('./component_activated_menu');
-const editableComponent = require('./editable_components');
-
-const SELECTOR_HINT = "fieldset > .govuk-hint";
-const SELECTOR_LABEL = "legend > :first-child";
-const SELECTOR_ITEM = ".govuk-radios__item";
-const SELECTOR_ITEM_HINT = ".govuk-hint";
-const SELECTOR_ITEM_LABEL = "label";
-
```

Comment:
And, with minor exceptions, this is the only way we should be bothered about using `const` keyword, people :-)